web/src/i18n/locales/en.yaml

```diff
@@ -92,21 +92,25 @@
       _no: "No"
     header: "Program Activities"
     namePrefix: "Activity"
-    namePrefixAndNum: "Activity {{number}}"
+    namePrefixAndNum: "Core Activity {{number}}"
     description: 
-      title: "Activity Description"
+      title: "Key Program Administration Information"
       helpText: "Summarize the needs of the activity for which requested funding will be used (e.g. administrative, outreach, auditing, Public Health Registry Development, etc)"
       reminder: "Note: This section is limited to 280 characters or less"
       summary: 
         subheader: "Brief Activity Summary"
-      detail: 
-        subheader: "Describe the activity in detail including images, diagrams, or other relevant information."
+      detail:
+        standard:
+          subheader: "Describe the activity in detail including images, diagrams, or other relevant information."
+        hie:
+          subheader: "Describe the activity in detail including images, diagrams, or other relevant information."
+          helpText: "The activity details must also include a description of three items: how the activity is linked to meaningful use; value proposition to providers, and sustainability plan describing how the activity will be sustained after HIE DDI funding is exhausted.	Additionally, include diagrams and descriptions of other HIE linkages as needed if they provide more context for how you'll complete the activity."
```

Comment:
Added the HIE-specific help text down here, next to the standard text.  Hopefully makes it easier to find.